type_lens/type_view.py

```diff
@@ -217,10 +217,10 @@ def is_subtype_of(self, typ: Any | tuple[Any, ...], /) -> bool:
             Whether the annotation is a subtype of the given type(s).
         """
         if self.origin:
-            if self.origin in UNION_TYPES:
+            if self.is_union:
                 return all(t.is_subtype_of(typ) for t in self.inner_types)
 
-            return self.origin not in UNION_TYPES and issubclass(self.origin, typ)
```

Comment:
this just seems to be dead code, since the above conditional would always have taken the other branch